website/content/docs/bloop-basics.md

```diff
@@ -0,0 +1,127 @@
++++
+toc = true
+weight = 2
+draft = false
+title = "The basics of bloop"
+date = "2018-02-09T10:15:00+01:00"
+description = "What is bloop and which problems does it solve?"
+bref = "Learn the ideas behind bloop, which problems solve and how it's designed."
++++
+
+## An overview of Bloop
+
+Bloop is a Scala command-line tool and build server developed at the Scala
+Center to make the compile and test feedback loop as tight as possible. This
+means a faster and more productive developer workflow.
+
+### A common pain point
+
+Edit, compile and test workflows are the bread and butter of our daily jobs.
+When our build is slow to respond, our productivity drops. Can Scala get as
+tight edit/compile/test workflows as in Clojure or OCaml? Bloop is a
+command-line tool and build server that brings you a tighter developer
+workflow. Slow Scala compile times can often be attributed to the slugishness
+of our build tool, and `bloop` finds a solution to them.
+
+We have created bloop to make you productive without getting in your way. We
+focus on how you can be faster at writing Scala code using your current build
+tool, whether it’s sbt, Maven or Gradle—without reinventing the wheel with a
+new build tool.
+
+### The main goal
+
+The primary goal of bloop is to keep your hot compilers around.
+
+A hot compiler is a long-running compiler that has been heavily optimized by
+JVM's [JIT](https://en.wikipedia.org/wiki/Just-in-time_compilation) and
+therefore reaches peak performance when compiling code. The difference
+between a cold and hot compiler varies. This variation is usually an order of
+magnitude large — a hot compiler can be between 10x and 20x faster than a
+cold compiler.
+
+Keeping hot compilers alive is crucial to be productive. However, it is a
+difficult task; compilers must outlive short- and long-running build
+processes, and there is no existing way to share them across different build
+tools and IDEs.
+
+Bloop is an agnostic build server that makes hot compiler accessible to all
+your toolchain — IDEs, build tools and terminals can share hot compilers and
+get peak performance at all times.
+
+By supporting popular build tools out of the box and different scenarios,
+`bloop` enforces by design best practices to be productive.
+
+### What bloop is not
+
+Bloop is **not** a new build tool, but rather a foundation block for Scala
+tooling. It can be used by Scala developers to be more productive and by
+tooling authors to write tools in a fast and reliable way.
+
+There is a lot of progress and research going into the build tools field.
+Whether you use established build tools like Bazel, Pants, sbt, Maven and
+Gradle or brand-new build tools like CBT and mill, you should benefit from
+the fastest of the workflows no matter what the internal architecture of your
+build tool is.
+
+### Components
```

Comment:
I'm afraid people won't read up to that point, and the information below is super important to use Bloop.

Reply:
If it's that important, maybe we should move it to the installation guide. It would be weird if we explain how bloop works without explaining what bloop does.